src/contents/shared/ProfileEditView.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { EmailSVG } from '../../components/svg/EmailSVG';
+import { NameInputSVG } from "../../components/svg/NameInputSVG";
+import { InputField } from "../../components/form/InputField";
+import { LocationSVG } from "../../components/svg/LocationSVG";
+import { BusinessCardSVG } from '../../components/svg/BusinessCardSVG';
+import { FileAttachmentSVG } from '../../components/svg/FileAttachmentSVG';
+import { CameraSVG } from '../../components/svg/CameraSVG';
+import PropTypes from 'prop-types';
+
+
+export const ProfileEditView = ({ updateUserHandler, name, photoURL, changeHandler, location, user, type, email, handleChangeUpload, progress, url }) => (
+
+	
+	<div className="section-content with-padding">
+
+		<form onSubmit={updateUserHandler}>
+			<h1>Edit your Profile: {user.displayName}</h1>
+			<InputField
+				svg={
+					<NameInputSVG classes="gb-icon gb-icon-medium gb-icon-white inputIcon" />
+				}
+				value={name}
+				changeHandler={changeHandler}
+				type="text"
+				name="name"
+			/>
+			<InputField
+				svg={
+					<EmailSVG classes="gb-icon gb-icon-medium gb-icon-white inputIcon" />
```

Comment:
The email, i don't think you want to give someone the power to change his email, so easily, most of the websites do not give you this possibility , changing the email is a big risk, so we do not have to deal with it for now, in my opinion

Reply:
Done!